### PR TITLE
AB#8585: Fix: show play and pricing buttons based on item type.

### PR DESCRIPTION
**Pull request by josejibin**
ADO card: ☑️ [AB#8585](https://dev.azure.com/S72/SHIFT72/_workitems/edit/8585)

Elab notes/AC: 📃 [Google Docs](https://docs.google.com)
- [ ] Has this been discussed with Delivery Team? 
Small change. Does this need an elab?

## Description of work

We do not want to show play and pricing buttons for all items. Show play button only if the item is film, season or episode and
show pricing buttons only if the item is film, season or bundle.  (Logic is copied from [cta_buttons.jet](https://github.com/shift72/core-template/blob/7683305458dd15cd4b81f7012d9f3e6e47a6e6a5/site/templates/common/cta_buttons.jet#L12))

This will help us to avoid the unwanted [`getAvailability` check](https://github.com/indiereign/shift72-web/blob/master/src/core/content/availability.service.js#L112) and thus no more the `slug_not_supported` errors.

## Edge cases/Caveats/Known issues
- N/A

## Config settings/Toggles required for the feature to work
- N/A

### Any PRs which this PR depends on
- N/A

### Any PRs dependent on this one
- N/A

### Affected Clients
 - All clients who use Kibble

## Checklist
- [ ] CI tests are passing Github actions (inc. linting)
- [ ] Key areas of the feature outlined for context and testing
- [ ] If there are designs for this work are they noted here and in the ADO card 
- [ ] Design review
- [ ] Have checked this at multiple screen resolutions and range of browsers
- [X] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [X] Updated changelog (if applicable)
- [ ] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/templates/items/meta_item.jet [hunk 1]
```diff
@@ -61,11 +69,17 @@
...
-              {{ if item.ItemType == "film" }}
+              {{if showPlayButton}}
+                <s72-play-button data-slug="{{item.Slug}}" title="{{item.Title}}"></s72-play-button>
+              {{end}}
+
+              {{if showPricingButtons}}
+                <s72-pricing-buttons data-slug="{{item.Slug}}" class="pricing-buttons-fit" title="{{item.Title}}"></s72-pricing-buttons>
+              {{end}}
+
+              {{if isFilm}}
```
Comment (additiverich) [inline site/templates/items/meta_item.jet:80]: Will this work for tv episodes (also watchable)

Comment (PeculiarGoat) [reply site/templates/items/meta_item.jet:80]: Yeah we only show the can-be-watched-button for films.

**Review by PeculiarGoat — APPROVED** [review]
Nice work!  LGTM!

I'm struggling to see how this change can be responsible for the error that @smitsanghvi21 mentioned though... seems unrelated to me.  Maybe we should get to the bottom of that before merging though!

**Comment by smitsanghvi21** [comment]
@josejibin I can see a new console error when I try to rent or buy a movie. Eg: RPM_Other TV Season 1 on https://staging-studio-kibble.shift72.com

Screen Shot 2022-06-13 at 11 09 31 AM

**Review by smitsanghvi21 — APPROVED** [review]

**Review by curtisadams28 — APPROVED** [review]

**Pull request merged**